HISTORY.md

```diff
@@ -0,0 +1,27 @@
+## 1.1.0
```

Comment:
Does this line also need a release date for the changelog generator to detect 1.1.0 as the last release?

Reply:
I can't see why it does. Github treats tags as releases by default. Not sure why it sees 0.1.0, but not 0.2.0, 1.0.0, or 1.1.0

Reply:
I think gcg parses the HISTORY.md to work out the `since_tag`.

Reply:
Do we need to add an Entry in HISTORY.md for 0.1.0?

Reply:
Not for gcg.  You only technically need to fix up the most recent entry.
`## 1.1.0 (2017-10-17)` should do it.